src/geo/leaflet/leaflet-cartodb-webgl-layer-group-view.js

```diff
@@ -0,0 +1,49 @@
+var TC = require('tangram.cartodb');
+var LeafletLayerView = require('./leaflet-layer-view');
+
+var LeafletCartoDBVectorLayerGroupView = window.L.TileLayer.extend({
+  includes: [
+    LeafletLayerView.prototype
+  ],
+
+  options: {
+    minZoom: 0,
+    maxZoom: 28,
+    tileSize: 256,
+    zoomOffset: 0,
+    tileBuffer: 50
+  },
+
+  events: {
+    featureOver: null,
+    featureOut: null,
+    featureClick: null
+  },
+
+  initialize: function (layerGroupModel, map) {
+    LeafletLayerView.call(this, layerGroupModel, this, map);
+    layerGroupModel.bind('change:urls', this._onURLsChanged, this);
+
+    this.tangram = new TC(map);
+
+    layerGroupModel.each(this._onLayerAdded, this);
+    layerGroupModel.onLayerAdded(this._onLayerAdded.bind(this));
+  },
+
+  onAdd: function (map) {
+
```

Comment:
If this is truly an `L.TileLayer` I would invoke `L.TileLayer.prototype.onAdd.call(this, map);` here.